sdk/src/main/scala/io/horizen/storage/leveldb/VersionedLDBKVStore.scala

```diff
@@ -133,9 +134,15 @@ final class VersionedLDBKVStore(protected val db: DB, keepVersions: Int) extends
   def versionIdExists(versionId: VersionId): Boolean =
     versions.exists(new ByteArrayWrapper(_) == new ByteArrayWrapper(versionId))
 
-  def getIterator(): StorageIterator = {
+  def getIterator: StorageIterator = {
     new DatabaseIterator(db.iterator())
   }
+
+  def recalculateVersionsToKeep: Unit = {
+    keepVersions = ConsensusParamsUtil.getConsensusSlotsPerEpoch(Option.empty) * 2 + 1
```

Comment:
let's simplify and remove recalculateVersionsToKeep : they will be always based on the max number of slots that are configured in all the forks, and set in the initialization

Reply:
Done